pkg/util/util_test.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"net"
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"k8s.io/klog"
+)
+
+func TestSocketPath(t *testing.T) {
+	// Create in the path where the test is run
+	socketPath := "reg.sock"
+	fileExists := false
+
+	// Test1: file does not exist. So clean up should be successful.
```

Comment:
Could you split each test case into its own function? Easier to see which has failed

Reply:
Changed to separate tests.